src/pages/guides/uxp_guide/uxp-misc/manifest-v5/index.md

```diff
@@ -0,0 +1,282 @@
+        "webview": {
+            "allow": "yes",
+            // domains --> string[] | "all"
+            "domains": [ "https://*.adobe.com", "https://*.google.com"],
+            "allowLocalRendering": "yes"
+        },
+        "launchProcess": {           
+            "schemes": 
+                [ "https", "slack"],            
+            "extensions": 
+                [ ".xd", ".ps" ],        
+        }, 
+    },
+}
+```
+
+## Changes to top-level metadata
+Starting from changes to top-level metadata, here are some keys that changed with the v5 upgrade. Read [manifest v4](../manifest-v4/) to learn more about each key/value field.
+
+Key path | Type | Description | Change 
+---------|------|-------------|-------
+`version` | string| Version number of your plugin in `x.y.z` format. | Plugins can specify semver format with no warnings. <br/>Specify at least one number and the minor and/or patch will be autofilled with zeroes.
+`title` | string | Title for the plugin. <br/>Defaults to plugin's name if none is specified | New in v5.
+`requiredPermissions` | object | Declare [plugin permissions](#plugin-permissions). | New in v5.
+`entrypoints` | `EntryPointDefinition[]` | Describes the entries your plugin adds to the _Plugins_ menu and plugin panel. | v5 changes in next section.
+
+## Changes to entry points
+The changes to `entryPoints` add flexibility in specifying the initial view/location of plugin panels.
+
+Key | Type | Description | Change
+----|------|------------ |------
+`id` | string | Unique identifier for the entry point. This `id` will also be mapped to entrypoints defined in your plugin code. | Passed as the id for the uxpcommand, uxpshowview (New V5) and  uxphideview (New V5) events.
+`hostUIContext` | object | Use [this property](#hostuicontext) to customize how entrypoints appear. | New in v5.
+`featureFlags` | object | Indicates feature flags in use. | `enableWebview` boolean flag. 
+
+### HostUIContext 
+By default, panels from the same plugin are grouped together by the host app. If more control is needed, developers can use this property.
+
+Key | Type | Default | Description
+----|------|------------|---
+`hideFromMenu` | boolean | `false` | If `true`, the entrypoint is hidden from the plugins menu.
+`hideFromPluginsPanel` | boolean | `false` | If `true`, the entrypoint is removed from the plugins panel (launchpad).
+`initialLocation` | string | plugin ID | Plugins are grouped together upon ID by default. You can specify a new ID to group subsets of panels together. 
+`triggerEvents` | string[] | -- | Specifies if the entrypoint should be triggered by host-determined events. 
+
+## Plugin Permissions
+Plugins using Manifest v5 will enjoy the enhancements in security with the introduction of new permissions model. Users will be asked for their consent when your plugin attempts to use openExternal, openPath, and sp-link/anchor tags. For everything else, consent is assumed given at install time.
```

Comment:
Let's put ticks around `openExternal`, `openPath`, and `sp-link`.

Remove "assumed".